src/meta/types/src/user_stage.rs

```diff
@@ -296,36 +298,76 @@ pub struct StageParams {
 
 #[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Eq, PartialEq)]
 pub enum OnErrorMode {
-    None,
```

Comment:
But we remove some fields from the OnErrorMode, if we store&serialize the old OnErrorMode, and deserialize with the new OnErrorMode, not sure if it works or not? cc @drmingdrmer

Reply:
> But we remove some fields from the OnErrorMode, if we store&serialize the old OnErrorMode, and deserialize with the new OnErrorMode, not sure if it works or not? cc @drmingdrmer

Changes to rust-types are always OK if it passes tests.
Only changes to protobuf-types may break compatibility.